styleguide/derek/pattern-components/leads/_lead.scss

```diff
@@ -1,9 +1,11 @@
 .lead-bodyWrapper {
   margin-bottom: 14px;
-}
 
-.lead-bodyWrapper > a {
-  @include standard-link;
+  a {
+    @include standard-link;
+    color: $rackspace-red;
+    text-decoration: underline;
```

Comment:
So I already approved this, but just realized `@standard-link` already provides the `color` and `text-decoration` so those two lines are unnecessary. 
https://github.com/rackerlabs/zoolander/blob/733794300d6bd0322e82f09af08d973c8f2312fe/styleguide/_themes/global/scss/fonts.scss#L37-L46